components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```
@@ -2545,14 +2545,65 @@ paths:
+
+  /mcp-servers/generate-from-openapi:
     post:
       tags:
         - MCP Servers
-      summary: Create an MCP server using the backend API's OpenAPI definition
+      summary: |
+        Create a MCP server using an OpenAPI definition.
       description: |
-        This operation can be used to create a MCP server using the backend API's OpenAPI definition. Provide 
-        either `url` or `file` to specify the definition.
+        This operation can be used to create a MCP server using the OpenAPI definition. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)